### ISSUE TITLE
Bundle the runtime with SQLQueryStress

### ISSUE THREAD

**Issue by buckleyGI** [ErikEJ/SqlQueryStress#146]
It took me some time to get SQLQueryStress running on our servers.
Nothing to do with the program itself but with the runtime.
When opening the program on the server it offered the following

```
---------------------------
SQLQueryStress.exe
---------------------------
To run this application, you must install .NET Core.

Would you like to download it now?
---------------------------
Yes   No   
---------------------------
```

However, pressing Yes did nothing. It could be that we are on a somewhat lockdowned server but it had internet access nonetheless.

After that I download the following as I though .NET 7 would be a suitable runtime.
Since that wasn't the case I downloaded the latest .NET Core 3.1 which reached end of life, meaning it is no longer supported

1. dotnet-sdk-7.0.302-win-x64
2. dotnet-sdk-3.1.426-win-x64

The last one worked.
A further complication on a server is that all these downloads are blocked so had to get the installers with a workaround (copy past via remote desktop) at their destination. 
Installing something on a server is also for some shops, especially a new runtime, a showstopper.

I now read in the Github readme (should have checked that first of course) that .NET 6 Desktop Runtime also suffices.

Bottom line, would it benefit if we pack the runtime with the tool so it is self contained and starts up immediately?
That's natively possibly these days with .NET without a hack as we did for .NET Framework.
it will increase the package size, not sure by how much. Haven't tried the feature and don't know if it does tree shaking.

Okay if I do a spike on this or are there reasons not to pursue this?

**Comment by ErikEJ** [comment]
Feel free to give it a shot! Unsure how that will work right now, as the tool is a .NET global tool

**Comment by buckleyGI** [comment]
Yes, I took the single exe as outputted to 
c:\Git\SqlQueryStress\src\SQLQueryStress\bin\Release\net6.0-windows\win-x64\publish\win-x64\
and deployed it to a server without  the necessary runtimes. It started up as expected.

**Comment by ErikEJ** [comment]
Very cool!

**Comment by ErikEJ** [comment]
I will do this in the build pipeline, thanks for doing the validation

**Comment by buckleyGI** [comment]
Your welcome. Glad to contribute back after getting something out of the tool.
Out of curiosity, where is the build pipeline hosted as I didn't find a trace of it in the repo or on this project.

**Comment by ErikEJ** [comment]
There should be a link to it in the pull request - it is hosted on appveyor  but no yaml file in the repo, sorry.
